code/game/objects/structures/dropship_equipment.dm

```diff
@@ -877,42 +927,356 @@
+	//Stops the pilot bringing up the rappel to prevent it being disabled
+	rappel_condition = RAPPEL_CONDITION_HOOKED
+	if(rappel_state == RAPPEL_STATE_RETRACTING)
+		rappel_state = RAPPEL_STATE_USABLE
+	update_icon_state()
+	X.balloon_alert_to_viewers("[X] tears at the rappel!","You start tearing up [src] to disable the host's sky-rope system!")
+	step(X, get_dir(X, src))
+	balloon_alert_to_viewers("The system starts visibly buckling...")
+	playsound(rope, 'sound/effects/grillehit.ogg', 50, TRUE)
+	playsound(src, 'sound/effects/grillehit.ogg', 50, TRUE)
+	if(!do_after(X, 5 SECONDS, TRUE, src, BUSY_ICON_DANGER, BUSY_ICON_HOSTILE))
+		rappel_condition = RAPPEL_CONDITION_GOOD
+		rappel_state = RAPPEL_STATE_USABLE
+		update_icon_state()
+		balloon_alert_to_viewers("The system stops buckling.")
+		return
+
+	playsound(rope, 'sound/effects/metal_crash.ogg', 50, TRUE)
+	playsound(rope, 'sound/effects/sparks1.ogg', 50, TRUE)
+	playsound(src, 'sound/effects/metal_crash.ogg', 50, TRUE)
+	playsound(src, 'sound/effects/creak1.ogg', 50, TRUE)
+	X.balloon_alert_to_viewers("[X] rips the rappel cord out from above, disabling it!","You have disabled their sky-rope!")
+	balloon_alert_to_viewers("You hear a horrible screeching sound as something breaks!")
+
+	var/datum/effect_system/spark_spread/sparks = new
+	sparks.set_up(5, 0, src)
+	sparks.attach(src)
+	sparks.start()
+	sparks.attach(rope)
+	sparks.start()
+
+	disabled_smoke = new(src, /particles/mecha_smoke)
+	disabled_smoke.particles.position = list(0,5,0)
+
+	rappel_condition = RAPPEL_CONDITION_DAMAGED
+	rappel_state = RAPPEL_STATE_RETRACTING
+	update_icon_state()
+	retract_rope()
+
+///Ghosts teleport to the rappel system when they click on the rope
+/obj/effect/rappel_rope/tadpole/attack_ghost(mob/dead/observer/user)
+	. = ..()
+	user.forceMove(get_turf(parent_system))
+
+///This is the rope that the system deploys, a subtype of the HvH deployment rappel.
+///Created by the rappel system on init and stored in the rappel system when it's not in use
+/obj/effect/rappel_rope/tadpole
+	icon = 'icons/Marine/mainship_props.dmi'
+	name = "tadpole rappel rope"
+	light_system = STATIC_LIGHT
+	light_power = 0.5
+	light_range = 2
+	///The rappel system this rope originates from
+	var/obj/structure/dropship_equipment/rappel_system/parent_system
+
+/obj/effect/rappel_rope/tadpole/Initialize()
+	. = ..()
+	if(istype(loc, /obj/structure/dropship_equipment/rappel_system))
+		parent_system = loc
+
+/obj/effect/rappel_rope/tadpole/update_icon_state()
+	. = ..()
+	if(parent_system.rappel_state == RAPPEL_STATE_IN_USE || parent_system.rappel_state == RAPPEL_STATE_RETRACTING)
+		icon_state = "rope_rappeling"
+		return
+	icon_state = "rope"
+
+///Going up the rappel. Going up retracts the rappel
+/obj/effect/rappel_rope/tadpole/attack_hand(mob/living/user)
+	. = ..()
+
+	if(parent_system.rappel_condition < RAPPEL_CONDITION_GOOD)
+		return
+
+	if(user.buckled_mobs)
+		//Drop anyone we're fireman carrying
```

Comment:
sad

Reply:
Tadpole should probably have to have land clear in order to rescue bodies